intertidal/exposure.py

```diff
@@ -445,8 +445,6 @@ def exposure(
         model=tide_model,
         times=time_range,
         directory=tide_model_dir,
-        ranking_points="data/raw/tide_correlations_2017-2019.geojson",
```

Comment:
Removed the link to this ranking file so it uses the default value here:  https://github.com/GeoscienceAustralia/dea-notebooks/blob/develop/Tools/dea_tools/coastal.py#L440